nameresolution/consul/consul_test.go

```diff
@@ -184,25 +184,19 @@ func (m *mockRegistry) get(service string) *registryEntry {
 }
 
 func TestInit(t *testing.T) {
-	t.Parallel()
```

Comment:
Why are we removing `Parallel`s here?

Reply:
`Parallel` makes it harder to read test results because the logs are full of `CONT`. As a helper, it makes sense when the tests are long-running and perhaps I/O bound, but these tests complete in nanoseconds anyways. We don't gain anything by making them parallel and we make reading test results harder.